src/views/Leaderboard.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from 'react';
-import './Styles.css';
+import './Leaderboard.css';
```

Comment:
Keep Styles.css. Talk to Samir. You're pretty much undoing all Samir's work for no reason. You aren't the one doing styles changes. Work with what you have instead of doing everything your way.